packages/wdio-spec-reporter/src/index.js

```diff
@@ -387,14 +387,16 @@ class SpecReporter extends WDIOReporter {
     getEnviromentCombo (caps, verbose = true, isMultiremote = false) {
         const device = caps.deviceName
         const browser = isMultiremote ? 'MultiremoteBrowser' : (caps.browserName || caps.browser)
-        const version = caps.version || caps.platformVersion || caps.browser_version
-        const platform = caps.os ? (caps.os + ' ' + caps.os_version) : (caps.platform || caps.platformName)
+        let version = caps.version || caps.browserVersion || caps.browser_version
+        let platform = caps.os ? (caps.os + ' ' + caps.os_version) : caps.platform
 
         // Mobile capabilities
         if (device) {
             const program = (caps.app || '').replace('sauce-storage:', '') || caps.browserName
             const executing = program ? `executing ${program}` : ''
-
+            version = caps.platformVersion
+            platform = caps.platformName
```

Comment:
Unsure if this should be: caps.platform || caps.platformName
Do you happen to know this?

Reply:
Basically platform is MAC (LINUX) and platformName is iOS (Android)
Checks these caps I got from BS.

### android:
```js
{
  platform: 'LINUX',
  webStorageEnabled: false,
  takesScreenshot: true,
  javascriptEnabled: true,
  databaseEnabled: false,
  networkConnectionEnabled: true,
  locationContextEnabled: false,
  warnings: {},
  systemPort: 8202,
  orientation: 'PORTRAIT',
  udid: '8CDX1MVLF',
  platformName: 'Android',
  platformVersion: '10',
  newCommandTimeout: 300,
  realMobile: 'true',
  enablePerformanceLogging: false,
  chromeOptions: {
    args: [
      '--proxy-bypass-list=<-loopback>',
      'test-type',
      '--proxy-server=http://194.165.185.56:26082',
      '--disable-features=TranslateUI'
    ]
  },
  loggingPrefs: { browser: 'INFO' },
  deviceName: '8CDX1MVLF',
  browserName: 'chrome',
  appPackage: 'com.android.chrome',
  appActivity: 'com.google.android.apps.chrome.Main',
  deviceUDID: '8CDX1MVLF',
  deviceApiLevel: 29,
  deviceScreenSize: '1080x2160',
  deviceScreenDensity: 440,
  deviceModel: 'Pixel 3',
  deviceManufacturer: 'Google',
  pixelRatio: 2.75,
  statBarHeight: 66,
  viewportRect: { left: 0, top: 66, width: 1080, height: 1962 }
}
```
### ios:
```
{
  webStorageEnabled: false,
  locationContextEnabled: false,
  browserName: 'safari',
  platform: 'MAC',
  javascriptEnabled: true,
  databaseEnabled: false,
  takesScreenshot: true,
  networkConnectionEnabled: false,
  platformVersion: '12.0',
  useXctestrunFile: true,
  bootstrapPath: '/usr/local/.browserstack/config/wda_derived_data_00008020-001164DE142A002E_1.14.0/Build/Products',
  orientation: 'PORTRAIT',
  'browserstack.isTargetBased': 'false',
  udid: '00008020-001164DE142A002E',
  'browserstack.use_w3c': 'true',
  'browserstack.networkLogs': 'true',
  'browserstack.console': 'INFO',
  'browserstack.local': 'false',
  appiumVersion: '1.80',
  device: 'iphone',
  os_version: '12',
  platformName: 'iOS',
  'browserstack.appium_version': '1.14.0',
  acceptSslCert: false,
  detected_language: 'webdriver/6.1.11',
  new_bucketing: true,
  browser: 'iphone',
  real_mobile: true,
  osVersion: '12',
  browser_name: 'iPhone',
  deviceName: 'iPhone XR',
  'browserstack.appiumVersion': '1.14.0',
  'browserstack.useW3C': 'true',
  sessionName: 'Xxx.ts sandbox',
  buildName: 'nick',
  projectName: 'nick',
  'browserstack.debug': 'false',
  'browserstack.tunnelIdentifier': '',
  'browserstack.appiumLogs': 'true',
  version: '',
  mobile: { browser: 'mobile', version: 'iPhone XR-12.1' },
  orig_os: 'ios',
  '64bit': false,
  'browserstack.video': 'true',
  'browserstack.video.disableWaterMark': 'true',
  proxy_type: 'node',
  realMobile: 'true',
  appium_port: 8082,
  automationName: 'XCUITest',
  safariInitialUrl: 'http://mobile-internet-check.browserstack.com',
  webkitResponseTimeout: 20000,
  deviceOrientation: 'portrait',
  'browserstack.ie.noFlash': 'false',
  acceptSslCerts: false,
  newCommandTimeout: 300,
  safariIgnoreFraudWarning: true,
  noReset: true,
  wda_port: 8402
}
```